src/modules/datepicker/services/calendar.service.ts

```diff
@@ -101,15 +101,15 @@ export class CalendarService {
         this.currentView = this.config.mappings.finalView;
 
         if (!this._selectedDate) {
-            let today = new Date().getTime();
+            let current = this.currentDate ? this.currentDate.getTime() : new Date().getTime();
```

Comment:
Set `currentDate` to `new Date()` in the constructor, and make this line just `this.currentDate.getTime()` since `currentDate` isn't nullable.